mantidproject/mantid
# Indirect - MSD Fit Plot Output has wrong x units

**Description of work.**
This PR introduces a Temperature (in Kelvin) label which is now used in `Indirect Data Analysis`->`MSDFit` interface.
This PR also adds documentation for several undocumented non-TOF convertible units.

**To test:**
1. `Interfaces`->`Indirect`->`Data Analysis`->`MSD Fit` Tab
2. Enter the data below
3. Click `Run` and wait
4. Click `Plot` and check that the x-axis label says `Temperature (K)`

**Data**
[osi104367-104371_graphite002_red_elwin_eq.zip](https://github.com/mantidproject/mantid/files/2495042/osi104367-104371_graphite002_red_elwin_eq.zip)
FitType: `Gaussian`
StartX: `0.2`
EndX: `0.9`

Fixes #23493 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
